src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -405,7 +211,7 @@ void Thumb_DragDelta(object sender, DragDeltaEventArgs e)
             GridViewColumnHeader header = senderAsThumb.TemplatedParent as GridViewColumnHeader;
             if ((header.Content as string) == Properties.Resources.ScannerResultControl_Thumb_DragDelta_Rule)
             {
-                HasUserResizedLvHeader = true;
+                this.listControl.HasUserResizedLvHeader = true;
```

Comment:
If `Thumb_DragDelta` is in the child, it seems like maybe `HasUserResizedLvHeader` would live completely in the child, as well

Reply:
Make complete sense to me after your last comment.
I'll move `HasUserResizedLvHeader` to the child as well as `DragDeltaEvent`